packages/databases-collections/src/components/collation-fields/collation-fields.jsx

```diff
@@ -1,13 +1,19 @@
+import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Select, Option, SelectSize } from '@mongodb-js/compass-components';
-import _ from 'lodash';
+import { css } from '@mongodb-js/compass-components';
+
+const optionsSelectDropdownStyles = css({
+  zIndex: 1,
+  'button:focus, button:focus-within': {
+    zIndex: 20
+  }
```

Comment:
I'm guessing this is some leafygreen select issue workaround? Might be worth adding a comment here

Reply:
It was just moved here from the deleted `collation-fields.module.less `file.